pom.xml

```diff
@@ -110,6 +110,7 @@
         <module>oidc-service-impl</module>
 		<module>consent-service-impl</module>
 		<module>vci-service-impl</module>
+		<module>oidc-ui</module>
```

Comment:
oidc-ui is not a pom project, its react application

Reply:
@ase-101 , Since the UI portion of esignet was not added in analysis before, we have added the module.anshul is adding test cases for it now.